api/foxpass_copy_group.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+"""
+This script requires the external libraries from requests
+pip install requests
+
+To run:
+python foxpass_copy_group.py --api-key <api_key> --source-group <group_name> --dest-group <group_name>
+"""
+import argparse
+import json
+import requests
+import sys
+
+URL = 'https://api.foxpass.com/v1/'
+ENDPOINT = 'groups/'
+API = URL + ENDPOINT
+
+def main():
+    args = get_args()
+    header = {'Authorization': 'Token ' + args.api_key}
+    source_list = get_group_list(header, args.source_group).json()['data']
+    dest_list = get_group_list(header, args.dest_group).json()['data']
+    update_list = compare_list(source_list, dest_list)
+    if len(update_list) < 1:
```

Comment:
style improvement to reduce un-necesssary indentation

if len(....) < 1:
   print("No users...)
   return

for user_name in update_list:
    put_group_member(...